static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -278,54 +269,21 @@ function SpanMetricsTable({
   filter: ModuleName;
   transaction: string | undefined;
 }) {
-  const location = useLocation();
-  const pageFilter = usePageFilters();
-
   // TODO: Add transaction http method to query conditions as well, since transaction name alone is not unique
-  const queryConditions = buildQueryConditions(filter || ModuleName.ALL, location);
-  if (transaction) {
-    queryConditions.push(`transaction = '${transaction}'`);
-  }
-
-  const query = getSpanListQuery(
-    pageFilter.selection.datetime,
-    queryConditions,
+  const {isLoading: areSpansLoading, data: spansData} = useSpanList(
+    filter ?? ModuleName.ALL,
+    transaction,
     'count',
     SPANS_TABLE_LIMIT
   );
 
-  const {isLoading: areSpansLoading, data: spansData} = useQuery<SpanDataRow[]>({
-    queryKey: ['spans', query],
-    queryFn: () => fetch(`${HOST}/?query=${query}&format=sql`).then(res => res.json()),
-    retry: false,
-    refetchOnWindowFocus: false,
-    initialData: [],
-  });
-
-  const groupIDs = spansData.map(({group_id}) => group_id);
-
-  const {isLoading: areSpansTrendsLoading, data: spansTrendsData} = useQuery<
-    SpanTrendDataRow[]
-  >({
-    queryKey: ['spansTrends'],
-    queryFn: () =>
-      fetch(
-        `${HOST}/?query=${getSpansTrendsQuery(pageFilter.selection.datetime, groupIDs)}`
-      ).then(res => res.json()),
-    retry: false,
-    refetchOnWindowFocus: false,
-    initialData: [],
-    enabled: groupIDs.length > 0,
-  });
-
   return (
     <SpansTable
       moduleName={ModuleName.ALL}
-      isLoading={areSpansLoading || areSpansTrendsLoading}
+      isLoading={areSpansLoading}
       spansData={spansData}
-      orderBy="count"
+      orderBy="-time_spent_percentage"
```

Comment:
hmm orderby isn't being applied to this request for some reason
<img width="375" alt="Screenshot 2023-06-08 at 7 46 26 AM" src="https://github.com/getsentry/sentry/assets/63818634/7c0eeee9-760d-4383-b113-763facec1f7c">